Martlet.schema

```diff
@@ -25,9 +25,8 @@ CREATE TABLE IF NOT EXISTS "Quotes" (
 	`Date`	TEXT
 );
 CREATE TABLE IF NOT EXISTS `Members` (
-	`ID`	      INTEGER UNIQUE,
-	`DisplayName` TEXT,
-	`Score`	      INTEGER
+	`ID`    INTEGER UNIQUE,
+	`Name`  TEXT
```

Comment:
ouch we'll need to do some weird stuff here

Reply:
Yeah. Like I mentionned in the PR description, we're gonna need to fix foreign keys which currently don't work, which will already break currency because users were added into it without being added to Members first, which need to be changed.

I have already built the new members table with all the current users and all users that ever got a reaction/reacted on a message, so when foreign keys are fixed, and once currency adds users to members first, we can also add the current users in the currency table to Members